website/default.twig

```diff
@@ -128,6 +128,15 @@
         <noscript><p><img src="http://mnapoli.piwikpro.com/piwik.php?idsite=2" style="border:0;" alt="" /></p></noscript>
         <!-- End Piwik Code -->
 
+        {% if preview %}
+            <!-- Livereload -->
+            <script>
+                document.write('<script src="http://' + (location.host || 'localhost').split(':')[0] +
+                        ':35729/livereload.js?snipver=1"></' + 'script>')
+            </script>
+            <!-- End Livereload -->
+        {% endif %}
```

Comment:
Oh I realize this is something we have to add to all templates. That means it won't work out of the box with anything except this repository. Maybe we could add it dynamically using a processing step? (after the html generation, we inject the script before the `</body>` tag)